src/index.ts

```diff
@@ -85,8 +85,13 @@ export interface ChartRendererConfigOptions extends DeprecatedConfigProperties,
     mode?: ChartRendererMode
     /**
      * Allows to toggle on or off some features of the cursor tooltip, displayed when hovering objects when using pointing devices like a mouse, or when tapping on an object on touch devices.
+     * @deprecated
```

Comment:
Maybe indicate which property to use instead of `objectTooltip`?

(same for the other deprecated properties)